slimeworldmanager-plugin/src/main/java/com/grinderwolf/swm/plugin/commands/sub/UnloadWorldCmd.java

```diff
@@ -58,5 +59,23 @@ public boolean onCommand(CommandSender sender, String[] args) {
 
         return false;
     }
+
+    @Override
+    public List<String> onTabComplete(CommandSender sender, String[] args) {
+        final List<String> toReturn = new LinkedList<>();
```

Comment:
I'd default the list object to null and then, when adding a world to it, check if it's null and create a new list in case it is. Then, when returning the list, you can return `Collections.emptyList()` if no worlds were found, avoiding the extra memory allocation when there are no matches.

Same goes with the other subcommands.

Reply:
Should be resolved with new commit, check it out please! :)